.travis.yml

```diff
@@ -20,10 +20,15 @@ notifications:
     if: branch = master AND type != pull_request
     on_failure: always
     recipients:
-      michael.haas01@sap.com
-      karthik.muthuswamy@sap.com
-      francesco.alda@sap.com
-      evgeny.arnautov@sap.com
+      # Michael
```

Comment:
@EvgenyArn This should be better!

Reply:
What's this? And, is this a hash of the email address?

Reply:
My question is more out of curiosity than a question :)

Reply:
@mhaas looks fancy, what is it? :)

Reply:
@EvgenyArn In Travis, each repo has an associated public and private key pair. I can encrypt data with the public key, and  Travis can decrypt it with the private key. This is also how credentials are handled for pypi upload and for the DAR service key used for the system tests :)